Allow Windows instances to reach Teamserver instances on ports 5000-5999

jsf9k opened the pull request:
## 🗣 Description ##

This pull request puts some security group rules in place to allow Windows instances to reach Teamserver instances on ports 5000-5999.

## 💭 Motivation and context ##

Resolves cisagov/cool-system-internal#127.

## 🧪 Testing ##

All automated tests pass.  I also applied these changes to env6 in our staging COOL environment and then verified that I could:
1. Run `yes | nc -l 5100` on a Teamserver instance
1. Point a web browser on the Windows instance to `http://teamserver0.env6`
1. See the HTML request from the web browser appear in the `nc` session on the Teamserver instance
1. Run `nc teamserver0.env6 5100` in a terminal on a Windows instance
1. View the many lovely `y\n`s that appear in the `nc` session on the Windows instance

In this way I verified that:
- The Windows instances can reach the Teamserver instances on the 5000-5999 port range
- Any responses sent by the Teamserver instances via ephemeral ports indeed reach the Windows servers

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] All new and existing tests pass.

jsf9k committed:
Allow Windows instances to hit Teamserver instances on ports 5000-5999

This was requested in cisagov/cool-system-internal#127.

mcdonnnj commented on windows_sg.tf:
```diff
@@ -74,3 +74,17 @@ resource "aws_security_group_rule" "windows_ingress_from_kali_instances" {
   to_port                  = 65535
 }
+
+# Allow egress from Windows to Teamserver instances on ports 5000-5999
+# (TCP only).  This port range was requested for use by assessment
+# operators in cisagov/cool-system-internal#127.
```
This aligns with the comment describing the `aws_security_group_rule.teamserver_ingress_from_windows_instances_via_5000_to_5999_tcp` resource.

```suggestion
# Allow egress from Windows instances to Teamserver instances on
# ports 5000-5999 (TCP only).  This port range was requested for use
# by assessment operators in cisagov/cool-system-internal#127.
```

mcdonnnj reviewed (approved):
One suggestion for consistency but otherwise LGTM ✔

jsf9k committed:
Tweak comment language

Make a comment's language more closely parallel the language structure of a similar comment.

Co-authored-by: Nick <50747025+mcdonnnj@users.noreply.github.com>

dav3r reviewed (approved):
No muss, no fuss 👍
